genai-perf/genai_perf/parser.py

```diff
@@ -269,6 +268,15 @@ def _check_load_manager_args(args: argparse.Namespace) -> argparse.Namespace:
     return args
 
 
+def _check_measurement_args(args: argparse.Namespace) -> argparse.Namespace:
```

Comment:
This logic needs to be replicated in the config as well otherwise what we do with the CLI isn't consistent with the config file.

Reply:
Thanks, added a new commit with the config changes and testing. Can you review it when you have a chance?